Assignment2/derbeire-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,46 @@
+{
+	"@context": "http://schema.org/",
+	"@type": "Person",
+    "@id": "http://example.com/resource/persona",
+	
+   "name":"Irene del Rincon Bello",
+   "hometown":"Madrid",
+   "age":"26",
+   "almaMater":"UPM",
+   "parents":{
+    "@type": "Person",
+     "@id": "http://example.com/resource/persona",
+      "name1":"Teresa",
+      "name2":"Alberto"
+   },
+   "subjects":{
+    "@type": "Subjects",
+     "@id": "http://example.com/resource/estudios",
+      "subject1":{
+         "@type": "Subject",
+        "@id": "http://example.com/resource/asignaturas/SemanticWeb",
+         "name":"Semantic Web",
+         "teachers":{
+            "@type": "Person",
+           "@id": "http://example.com/resource/profesores/SemanticWeb",
+            "teacher1":"Oscar Corcho",
+            "teacher2":"Paola Espinoza",
+            "teacher3":"Raul Garcia"
```

Comment:
You can define an array of strings to define the professor names and only use one attribute -> e.g. "teachers":[the list of professor names]. You also should to delete "@id" and "@type".
But, if you want to define them as objects, or as Persons you have to define them as separate sets and add "@type":"Person" for each one.